26-register_publication/274-creator_navigates_to_the_reference_tab_and_selects_contribution_to_journal.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Creator navigates to the Reference tab and selects Contribution to Journal
+
+  @274
+  Scenario: Creator navigates to the Reference tab and selects Contribution to Journal
+    Given that the Creator opens their Publication and navigates to the Reference tab
+    When they select Contribution to Journal as Reference Type
+    And they see a dropdown of potential Reference Subtypes consisting of:
+      | Journal article      |
+      | Short communication  |
+      | Letter to the Editor |
+      | Book review          |
+      | Editoral             |
+      | Corrigendum          |
```

Comment:
Corrigendum burde kanskje ikke vært her, siden hele den biten dekkes av story 1625 (siden den har flere ulikheter ift de andre subtypene som ligger her)..?

Reply:
Helt greit å splitte velging av type som egen story og legge på storier for hver subtype som ikke har like felter